libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/ForEach.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Actions
     /// <summary>
     /// Executes a set of actions once for each item in an in-memory list or collection.
     /// </summary>
+    [Obsolete("Use ForEachIterative instead.")]
```

Comment:
did you mean "Use ForEachElement instead."

Reply:
Ha, yes, I renamed a few times, let me update